src/presentation/event-handlers/Issues.ts

```diff
@@ -131,14 +131,16 @@ export class IssuesEventHandler implements IIssueEvent {
 
   commentEdited(): HandlerFunction<"issue_comment.edited"> {
     return (event) => {
-      const body = markdownToHTML(event.payload.issue?.body ?? "");
+      const oldBody = markdownToHTML(event.payload.changes!.body.from ?? "");
```

Comment:
It's safer to use `?.` instead

```suggestion
      const oldBody = markdownToHTML(event.payload.changes?.body.from ?? "");
```